src/olympia/accounts/views.py

```diff
@@ -487,3 +490,56 @@ def post(self, request):
             'fxa_id': user.fxa_id,
             'session_cookie': cookie,
         }, status=201)
+
+
+class AccountNotificationViewSet(ListModelMixin, GenericViewSet):
+    """Returns account notifications.  If not already set by the user defaults
```

Comment:
More double whitespaces, also the style is a bit wrong, usually it's something along the lines of

```python
"""Returns account notifications.

If not already set by the user, defaults will be returned.
"""
```

According to pep08 and other style guides (imho)